INSTALL.rst

```diff
@@ -308,13 +308,16 @@ You should also set apache to start on boot::
 
   sudo chkconfig httpd on
 
-The networking server may be started by running::
+The networking server may be started by running the following as the HaaS user::
 
   haas serve_networks &
```

Comment:
Yeah, I was figuring that on reboot the service would take over and this would allow the person going through the install to at least see if serve_networks barfed at all, but in reality the service definition would be best. I'll add that a bit later.